src/prisma/__mocks__/prisma.service.mock.ts

```diff
@@ -1,7 +1,9 @@
 import { faker } from '@faker-js/faker';
-import { Prisma, Recipe, User } from '@prisma/client';
+import { Prisma, Recipe, User, Role } from '@prisma/client';
 import { UpdateRecipeRequest } from 'src/recipe/dto';
 
+let userId: number;
```

Comment:
Its main purpose is for e2e updateRecipe test. To pass this test, `user id` has to match recipe's `author id`, because either admin or **owner** can update it. Therefore I sticked with this variable, which is being updated every time `user.findUnique` gets called, so the `recipe.findUnique` will mock a recipe with the same `authorId` as user's `id'